calico-enterprise/reference/component-resources/node/felix/configuration.mdx

```diff
@@ -109,6 +109,18 @@ The full list of parameters which can be set is as follows.
 	* `enabled` - when CTLB is turned off make sure that services are always accessible.
 	* `udp` - turns off CTLB for UDP only and makes sure that services are always accessible. Preferred setting to make sure that DNS works.
 
+### Go runtime configuration
+
+These configuration pararmeters allow configuration of the Go language runtime.
+They allow the `GOGC`, `GOMEMLIMIT` and `GOMAXPROCS` tunables to be accessed
+via Felix's configuration.
+
+| Configuration file parameter        | Environment variable                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Schema  |
+| ----------------------------------- | ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | 
+| GoGCThreshold                       | `FELIX_GOGCTHRESHOLD`, `GOGC`             | Sets the Go runtime's garbage collection threshold; this is the percentage that the heap is allowed to grow before GC is triggered.   In general, doubling the value halves the CPU timespent doing GC, but it also doubles peak GC memory overhead.  A special value of -1 can be used to disable GC entirely; this should only be used in conjunction with the `GoMemoryLimitMB` setting.  This setting is overridden by the `GOGC` environment variable.                                                                                                                                                        | int     |
```

Comment:
GoGCThreshold --> ``GoGCThreshold``

Reply:
And for the other parameters in this column, to be consistent with the rest of the page.

Reply:
Of course, not the entire page adheres to this norm. But generally I prefer to keep parameters in code font.